wiring/src/spark_wiring_ble.cpp

```diff
@@ -1135,20 +1135,33 @@ size_t BleCharacteristic::description(char* buf, size_t len) const {
 }
 
 ssize_t BleCharacteristic::setValue(const uint8_t* buf, size_t len) {
-    if (buf == nullptr || len == 0) {
+    if (buf == nullptr || len == 0 || len > BLE_MAX_ATTR_VALUE_PACKET_SIZE) {
```

Comment:
I think I suggested doing this only for `is_integral` case. This is still useful for buffers and strings to have streaming (e.g. UART over BLE).